db/migrate/20190703120648_add_foreign_key_to_site_and_recruitment_cycle.rb

```diff
@@ -0,0 +1,5 @@
+class AddForeignKeyToSiteAndRecruitmentCycle < ActiveRecord::Migration[5.2]
+  def change
+    add_foreign_key :site, :recruitment_cycle, column: :recruitment_cycle_id
```

Comment:
why is the one explicit with `column` ???

Reply:
Has no effect as its the default so will remove.